### PR TITLE
Error handling

### LINKED ISSUE #53: User should become aware of an error in more than one way
For example, in `FindPark`, a `<p>` element prompts the user to enter valid criteria if the input is empty on submit.

- [ ] Add styling so that the error message stands out in color
- [ ] Add highlight on empty input to indicate where error comes from

### PR DESCRIPTION
**Pull request by stacyp2006**
### Participating Group Members:

- Stacy Potten

### Code Highlights:

- Adds styling to error message and input field when input field is empty
- Disables `Find parks near me` button when geolocation is inactive

### Have Tests Been Added?

- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?
- The `:class` feature in Vue is really handy for toggling between classes

### Issues:

- Addresses #
- Closes #53, #58 

### Screenshots (if appropriate):

\*

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [ ] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
Thank you for the error handling! Very appreciated to have some extra feedback for our users.

File: src/components/FindPark.vue [hunk 1]
```diff
@@ -1,23 +1,31 @@
 <template>
   <section>
     <h2>Let's Go Play!</h2>
-    <button @click="searchByLocation">Find a dog park near me!</button>
+    <button
+      :class="{ disabled: !this.$store.state.geolocation }"
```
Comment (bretm9) [inline src/components/FindPark.vue:5]: This is awesome! `:class` seems really flexible. I'll definitely be using this on my branch.

**Review by bretm9 — APPROVED** [review]
Great additions. Making the "near me" button disabled until geolocation data comes in not only makes for a more fool-proof UX but also makes our site easier to test since we don't have to guess how long we need to wait before clicking the button while manually testing things. Thanks for this!

**Review by nicolegooden — APPROVED** [review]
Looks great! Thanks for taking care of this and knocking some items off of our backlog.
Will merge now.

**Pull request merged**